bird.js

```diff
@@ -26,12 +26,17 @@ class Bird {
   }
 
   up() {
-    this.velocity += this.lift;
+    if (this.delay)
+        this.delay = false;
+
+    if (!this.delay) {
+        this.velocity = -7;
```

Comment:
Creating a global variable for the `-7` would be a good thing to do. It's easier to modify static values when they are global.